hivemind/optim/collaborative.py

```diff
@@ -197,6 +200,8 @@ def load_state_from_peers(self, **kwargs):
                 try:
                     self.averager.load_state_from_peers(timeout=self.load_state_timeout, **kwargs)
                     break
+                except KeyboardInterrupt:
+                    raise
```

Comment:
This would previously cause a deadlock if user would KeyboardInterrupt CollaborativeOptimizer during load_state_from_peers

Reply:
Maybe replace `except BaseException` -> `except Exception` instead of this change?

Reply:
Sometime in future, yes, but currently i'm afraid that it may accidentally forward CancelledError or something like that.

Reply:
Okay